news/12308.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+When using the ``importlib.metata`` backend (the default on Python 3.11+), ``pip
+list`` does not show installed egg distribution more that once anymore.
```

Comment:
```suggestion
When using the ``importlib.metadata`` backend (the default on Python 3.11+), ``pip
list`` does not show installed egg distributions more that once anymore.
```

In addition to Paul's typo find :slightly_smiling_face:

Reply:
So I guess I hold the record of typos per sentence :upside_down_face: